java/ql/src/Security/CWE/CWE-022/TaintedPath.ql

```diff
@@ -44,9 +49,7 @@ class TaintedPathConfig extends TaintTracking::Configuration {
   }
 }
 
-from DataFlow::PathNode source, DataFlow::PathNode sink, PathCreation p, TaintedPathConfig conf
-where
-  sink.getNode().asExpr() = p.getAnInput() and
-  conf.hasFlowPath(source, sink)
-select p, source, sink, "$@ flows to here and is used in a path.", source.getNode(),
+from DataFlow::PathNode source, DataFlow::PathNode sink, TaintedPathConfig conf
+where conf.hasFlowPath(source, sink)
+select sink, source, sink, "$@ flows to here and is used in a path.", source.getNode(),
```

Comment:
Will this change in the `select` columns dismiss all existing alerts and create new ones for projects that already have results?

Reply:
I suppose so -- I can have this report a `Top` so I can preserve the existing reporting location

Reply:
This query needs an extensive refactor anyway, so I'm not against doing that for the time being. I think it's better than cause friction with all the alert changes.

Reply:
Amended as suggested